subjects/forum/forum.audit.en.md

```diff
@@ -0,0 +1,143 @@
+
+###### Does it present the changes made in the logged in browser?
+
+###### Is it asked in the register for an email and a password?
+
+###### Does the project detects if the email or password are wrong?
+
+###### Does the project detects if the email or user name is already taken in the register?
+
+#### Docker
+
+###### Does the project have Dockerfiles?
+
+##### Try to run the command `"docker image build [OPTINS] PATH | URL | -"` to build the image using using the project Dockerfiles?
+```
+student$ docker images
+REPOSITORY              TAG                             IMAGE ID            CREATED             SIZE
+<name of the image>     latest                          85a65d66ca39        7 seconds ago       795MB
+```
+###### Run the command `"docker images"` to see images. Does all images build as above?
+
+##### Try running the command `"docker container run [OPTIONS] IMAGE [COMMAND] [ARG...]"` to start the containers using the images just created.
+```
+student$ docker ps -a
+CONTAINER ID        IMAGE                  COMMAND                  CREATED             STATUS              PORTS                    NAMES
+cc8f5dcf760f        ascii-art-web-docker   "./server"               6 seconds ago       Up 6 seconds        0.0.0.0:8080->8080/tcp   ascii-art-web
+```
+###### Run the command `"docker ps -a"` to see containers. Is the docker containers running as above?
+
+##### Try running the comment `"docker network ls"`
+```
+NETWORK ID          NAME                DRIVER              SCOPE
+24560f95a216        <network-name>      bridge              local
+24560f95a216        bridge              bridge              local
+3f12239adb4f        host                host                local
+159cdc8d8083        none                null                local
+```
+###### Is the network a bridge driver and does it appear in the list of networks as above?
+
+##### Try openning a shell to the running container, using the [command](https://docs.docker.com/engine/reference/commandline/exec/) `"docker exec [OPTIONS] CONTAINER COMMAND [ARG...]"` (ex: `"docker exec -it <container-name> /bin/bash"`) and ping the other container that are connected to the same network, for [example](https://docs.docker.com/engine/tutorials/networkingcontainers/).
+
+###### Did it ping with success? Are the multiple services/containers communicating between each other?
+
+###### Does the project present no [unused object](https://docs.docker.com/config/pruning/)?
+
+###### Did the server behaved as expected?(did not crashed)
+
+###### Does the server use the right [HTTP method](https://developer.mozilla.org/en-US/docs/Web/HTTP/Methods)?
+
+###### Has the website runned without crashing at anytime?
+
+###### Are all the pages working? (Absence of 404 page?)
+
+###### Does the project avoid [HTTP status 400](https://kinsta.com/knowledgebase/400-bad-request/#causes)?
```

Comment:
Confusing, what do you mean ?

Reply:
What I meant by that is to see if in the project  they are handling bad requests (HTTP status 400) and internal server errors (HTTP status 500).

Reply:
It would then be preferable to speak of "handle" and not "avoid", as some Alem students told us, "avoid" can be understood in this way: "make sure it's impossible to get that HTTP status"

Reply:
ok, so i will change it to handle, thanks